SRC/PORT_IMAGE.H

```diff
@@ -0,0 +1,11 @@
+#ifndef ULTIMATETAPANKAIKKI_PORT_IMAGE_H
+#define ULTIMATETAPANKAIKKI_PORT_IMAGE_H
+namespace tk_port
+{
+    char *replace_extension_with_png( const char *name );
```

Comment:
Minor nitpick but in other files namespaces aren't indented (same on .src).

Reply:
Yeah, blame CLion. 😅